internal/services/network/application_gateway_resource.go

```diff
@@ -4218,11 +4228,15 @@ func expandApplicationGatewaySslProfiles(d *pluginsdk.ResourceData, gatewayID st
 
 		name := v["name"].(string)
 		verifyClientCertIssuerDn := v["verify_client_cert_issuer_dn"].(bool)
+		verifyClientCertificateRevocation := v["verify_client_certificate_revocation"].(string)
 
 		output := network.ApplicationGatewaySslProfile{
-			Name: utils.String(name),
+			Name: pointer.To(name),
 			ApplicationGatewaySslProfilePropertiesFormat: &network.ApplicationGatewaySslProfilePropertiesFormat{
-				ClientAuthConfiguration: &network.ApplicationGatewayClientAuthConfiguration{VerifyClientCertIssuerDN: utils.Bool(verifyClientCertIssuerDn)},
+				ClientAuthConfiguration: &network.ApplicationGatewayClientAuthConfiguration{
+					VerifyClientCertIssuerDN: pointer.To(verifyClientCertIssuerDn),
+					VerifyClientRevocation:   network.ApplicationGatewayClientRevocationOptions(verifyClientCertificateRevocation),
```

Comment:
If no value is provided for `verify_client_certificate_revocation` then we will be sending an empty string here? Shouldn't we be defaulting this to `None` if this isn't set?

Reply:
behaviorally it is same, the API return `None` even if it is not set.